scripts/item/item-sheet.js

```diff
@@ -174,6 +174,8 @@ class ItemSheetWfrp4e extends ItemSheet
 
     data.showBorder = data.item.img == "systems/wfrp4e/icons/blank.png" || !data.item.img
     data.isGM = game.user.isGM;
+    data.isOwned = this.item.isOwned;
+    data.flags = this.item.data.flags;
```

Comment:
Pretty sure these can be accessed already, have you investigated that?

Reply:
Yes, I mean I tried to find them by looking at the 'data' variable in chrome inspector but I could have missed them if they are hidden deeper than I explored. I'll take another look tonight

Reply:
I could not find isOwned (it's a getter, not actual item data), but inside `data` is `item.flags`, so you can access that in handlebars with just `{{item.flags}}`